### PR TITLE
YLP-2218 Use selected team in Remove patient dialog

### PR DESCRIPTION
**Pull request by celia-rouis**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx [hunk 1]
```diff
@@ -84,6 +93,21 @@ describe('HCP home page', () => {
...
+    const removeDialog = screen.getByRole('dialog')
+
+    const dialogTitle = within(removeDialog).getByText(`Remove ${monitoredPatient.profile.firstName} ${monitoredPatient.profile.lastName} from My private practice`)
+    expect(dialogTitle).toBeVisible()
+    const dialogQuestion = within(removeDialog).getByTestId('modal-remove-patient-question')
+    expect(dialogQuestion).toHaveTextContent(`Are you sure you want to remove ${monitoredPatient.profile.firstName} ${monitoredPatient.profile.lastName} from My private practice?`)
+    const dialogInfo = within(removeDialog).getByText('You will no longer have access to their data.')
+    expect(dialogInfo).toBeVisible()
+    const confirmRemoveButton = within(removeDialog).getByRole('button', { name: 'Remove patient' })
+    expect(confirmRemoveButton).toBeVisible()
```
Comment (TimCanu) [inline packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx:108]: **❔ question:** ‏Shouldn't you click on the confirm button and check that the HTTP call to remove a patient has been made?

File: packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx [hunk 2]
```diff
@@ -104,11 +128,24 @@ describe('HCP home page', () => {
...
+
+    const title = within(removeDialog).getByText(`Remove ${unmonitoredPatient.profile.fullName} from ${teamThree.name}`)
+    expect(title).toBeVisible()
+    const question = within(removeDialog).getByTestId('modal-remove-patient-question')
+    expect(question).toHaveTextContent(`Are you sure you want to remove ${unmonitoredPatient.profile.fullName} from ${teamThree.name}?`)
+    const info = within(removeDialog).getByText('You and the care team will no longer have access to their data.')
+    expect(info).toBeVisible()
+    const alertInfo = within(removeDialog).getByText('If you want to remove the patient from another care team, you must first select the care team from the dropdown menu at the top right of YourLoops.')
+    expect(alertInfo).toBeVisible()
+    const cancelButton = within(removeDialog).getByText('Cancel')
```
Comment (TimCanu) [inline packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx:143]: Same question

File: packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx [hunk 3]
```diff
@@ -144,6 +181,7 @@ describe('HCP home page', () => {
   })
 
   it('should allow to remove a patient who is in multiple teams', async () => {
```
Comment (TimCanu) [inline packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx:183]: I don't think that this test is relevant anymore (as we cannot choose the team from the dropdown anymore)

File: packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx (hunk 2 above)
Comment (celia-rouis) [reply packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx:143]: The click on `confirmRemoveButton` is done in this test, see lines 146-157

Comment (TimCanu) [reply packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx:153]: I am but a blind boi :dark_sunglasses:

Comment (celia-rouis) [reply packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx:153]: :man_with_probing_cane:

**Review by TimCanu — APPROVED** [review]

**Pull request merged**